CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1215,16 +1215,12 @@ local function CreateUnreadMessagesNotifier(ChatModule)
 	return chatCounter
 end
 
-local useNewBubbleChatSuccess, useNewBubbleChatEnabled = pcall(function() return settings():GetFFlag("CorescriptNewBubbleChatEnabled") end)
-useNewBubbleChatEnabled = useNewBubbleChatEnabled and useNewBubbleChatSuccess
-
 local function CreateChatIcon()
 	local chatEnabled = game:GetService("UserInputService"):GetPlatform() ~= Enum.Platform.XBoxOne
 	if not chatEnabled then return end
 
 	local ChatModule = require(GuiRoot.Modules.ChatSelector)
 
-	local bubbleChatIsOn = not PlayersService.ClassicChat and PlayersService.BubbleChat
```

Comment:
ChatModule:IsBubbleChatOnly() does pretty much the same thing, but it accounts for the properties being able to be set in the ChatSettings module.